File: BookingApp/src/main/java/com/example/BookingApp/users/model/DeleteAccountRequest.java
package com.example.BookingApp.users.model;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import org.hibernate.annotations.GenericGenerator;

import javax.persistence.*;

@Entity
@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class DeleteAccountRequest {
    @Id
    @GeneratedValue(strategy = GenerationType.AUTO, generator="seq")
    @GenericGenerator(name = "seq", strategy="increment")
    private Long id;
    private String description;
    private boolean approved;
    @OneToOne(fetch = FetchType.LAZY,cascade = CascadeType.ALL)
    private Client client;
}

File: BookingApp/frontend/src/app/shared/models/DeleteAccountRequest.ts
import { Client } from "./Client";

export class DeleteAccountRequest{
    public id : Number;
    public description : String;
    public approved : boolean ;
    public clientId : Number;
    public client : Client;
    public solved : boolean;

    constructor(id:Number, description : String, approved : boolean, clientId : Number){
        this.id = id;
        this.description = description;
        this.approved = approved;
        this.clientId = clientId;
    }

}